res_users_password_security/README.rst

```diff
@@ -0,0 +1,94 @@
+.. image:: https://img.shields.io/badge/license-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
```

Comment:
be careful: be consistent in your licensing. Everywhere in your headers you state LGPL